### ISSUE TITLE
How do you compile the source and Revisions?

### ISSUE THREAD

**Issue by GoogleCodeExporter** [facelift/mupen64gc#170]
```
?
```

Original issue reported on code.google.com by `Gamewiza...@gmail.com` on 9 Nov 2009 at 2:15

**Comment by GoogleCodeExporter** [comment]
```
Gamewizard71,

The current changes to source you see us updating almost daily on here are not 
the
latest changes. It is just the migration from our private svn code base (before 
beta
1) to the public repo here. I suggest you use the bundled sourcecode available 
here
at the downloads page for this project as it is the most up to date available 
to the
public.

You'll need to use devkitPPC r17 and libOGC 1.7.0 to compile it as we cannot 
ensure
that it will build with anything later than that (that's what it was compiled 
with).

To sum things up, what you build will be identical to what we released if you 
build
from the bundled .zip of sourcecode on our downloads page, but if you build 
from the
latest source here, it will be more outdated (as of now).
```

Original comment by `emuki...@gmail.com` on 9 Nov 2009 at 4:47
- Changed state: **WontFix**
- Added labels: **Priority-Low**, **Type-Other**
- Removed labels: **Priority-Medium**, **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
-_-' ... thanks
```

Original comment by `Gamewiza...@gmail.com` on 9 Nov 2009 at 10:18
